### PR TITLE
Fix issue #115

### LINKED ISSUE #115: 🐛 [Bug]: Execute the pnpm build:ui command to build the error report
### Version

latest

### Vue Version

latest

### Link to minimal reproduction

执行以下命令构建组件库产物报以下错：
```
"./pc.vue" is imported as an external by "../../packages/vue/src/button-group/src/index.ts", but is already an existing non-external module id.
```

### Step to reproduce

执行以下命令构建组件库产物报以下错：
```
"./pc.vue" is imported as an external by "../../packages/vue/src/button-group/src/index.ts", but is already an existing non-external module id.
```

### What is expected

不报错

### What is actually happening

报错

### Any additional comments (optional)

_No response_

### PR DESCRIPTION
**Pull request by acyza**
# PR

## PR Checklist

Please check if your PR fulfills the following requirements:

- [x] The commit message follows our [Commit Message Guidelines](https://github.com/opentiny/ui-vue/blob/main/CONTRIBUTING.md)
- [x] Tests for the changes have been added (for bug fixes / features)
- [x] Docs have been added / updated (for bug fixes / features)

## PR Type

What kind of change does this PR introduce?

- [x] Bugfix
- [ ] Feature
- [ ] Code style update (formatting, local variables)
- [ ] Refactoring (no functional changes, no api changes)
- [ ] Build related changes
- [ ] CI related changes
- [ ] Documentation content changes
- [ ] Other... Please describe:

## What is the current behavior?

Issue Number: N/A

## Does this PR introduce a breaking change?

- [ ] Yes
- [x] No

## Other information

fix #115

### CODE REVIEW

**Comment by acyza** [comment]
windows,linux,macos都测了下。https://github.com/acyza/tiny-vue/actions/runs/4642487840/jobs/8216432804

**Comment by kagol** [comment]
> windows,linux,macos都测了下。https://github.com/acyza/tiny-vue/actions/runs/4642487840/jobs/8216432804

@acyza 我尝试了下确实可以了，想了解下这两处修改实际上是哪个修复了再macOS下的构建报错问题呢？另外将`.vue`改成`.js`会不会有别的影响呢？

File: internals/cli/src/commands/build/build-ui.ts [hunk 1]
```diff
@@ -63,33 +64,33 @@ export const getBaseConfig = ({ vueVersion, dtsInclude, dts, buildTarget, themeV
...
+          tsConfigFilePath: `tsconfig.vue${vueVersion}.json`,
+          aliasesExclude: [/@opentiny\/vue.+/],
+          compilerOptions: {
+            paths: {
+              ...compilerOptions.paths,
+              // 一定要映射到 packages/vue 下对应的 vue 版本和 @vue/composition-api 才能正确生成 dts
+              'vue': [`packages/vue/node_modules/vue${vueVersion}`],
+              '@vue/runtime-core': ['packages/vue/node_modules/@vue/runtime-core'],
+              '@vue/runtime-dom': ['packages/vue/node_modules/@vue/runtime-dom'],
+              '@vue/composition-api': ['packages/vue/node_modules/@vue/composition-api']
```
Comment (acyza) [inline internals/cli/src/commands/build/build-ui.ts:77]: 这段仅eslint调整了格式，无修改

**Review by kagol — APPROVED** [review]

File: internals/cli/src/commands/build/build-ui.ts (hunk 1 above)
Comment (kagol) [reply internals/cli/src/commands/build/build-ui.ts:77]: > 这段仅eslint调整了格式，无修改

这段评论很贴心了👍

**Comment by hetianxiang** [comment]
it's hava a problem in component file, we can see other file like '../../package'  path in packages dist2 or dist3

**Pull request merged**